server/src/test/java/org/eclipse/lsp/cobol/usecases/engine/UseCaseUtils.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Broadcom.
+ * Copyright (c) 2021 Broadcom.
```

Comment:
Why?

Reply:
No idea, seems like some automatic update. Should it be 2020?

Reply:
I think that it must be the year or file creation.